app/helpers/drawings_helper.rb

```diff
@@ -12,30 +12,41 @@ def form_title
   end
 
   def radio_statuses
-    # Example output: [ ["status1", "status1"], ["status2", "status2"] ]
-    [].tap do |arr|
-      Drawing.statuses.keys.each { |s| arr << [s, s] }
-    end
+    get_humanized_options(Drawing.statuses.keys)
   end
 
   def radio_genders
-    # Example output: [ ["gender0", "Gender 0"], ["gender1", "Gender 1"] ... ]
-    [].tap do |arr|
-      Drawing.genders.keys.each { |s| arr << [s, s.humanize] }
-    end
+    get_humanized_options(Drawing.genders.keys)
   end
 
   def mood_select_box
     # Example output: [ ["1 (sad face)", 1], ["2", 2] ... ["4", 4] ["5 (happy face)", 5] ]
-    selections = [].tap do |arr|
-      (1..5).each { |n| arr << [n.to_s, n] }
+    selections = (1..5).map do |number| 
+      [number.to_s, number]
     end
+
     selections[0][0] += " 😢"
     selections[-1][0] += " 😃"
     selections
   end
 
+  def journey_options
```

Comment:
As `Drawing::STAGES_OF_JOURNEY` is now an array of strings already in "humanized" form, we could actually just call `Drawing::STAGES_OF_JOURNEY`in the view to avoid having to call the humanize method unnecessarily:

```ruby
# views/drawings/_form.html.haml
.form-group
  = f.input :stage_of_journey, collection: Drawing::STAGES_OF_JOURNEY, label: 'Stage of journey', required: false
```

We could then lose this method altogether